packages/market-service/src/coingecko/coingecko.ts

```diff
@@ -122,11 +112,11 @@ export class CoinGeckoMarketService implements MarketService {
       Also a lot of time when max_supply is null, total_supply is the maximum supply on coingecko
       We can reassess in the future the degree of precision we want on that field */
       return {
-        price: marketData.current_price?.[currency].toString(),
-        marketCap: marketData.market_cap?.[currency].toString(),
-        changePercent24Hr: marketData.price_change_percentage_24h,
-        volume: marketData.total_volume?.[currency].toString(),
-        supply: marketData.circulating_supply.toString(),
+        price: bnOrZero(marketData?.current_price?.[currency]).toString(),
```

Comment:
There is already a null check for `marketData` above so we don't need the optional